e2e-tests/functions

```diff
@@ -240,8 +259,23 @@ wait_for_delete() {
 deploy_pmm_server() {
 	local platform=kubernetes
 	helm uninstall -n "${NAMESPACE}" pmm || :
-	helm install monitoring -n "${NAMESPACE}" --set imageTag=${IMAGE_PMM_SERVER#*:} --set service.type="LoadBalancer" \
-		--set imageRepo=${IMAGE_PMM_SERVER%:*} --set platform="${platform}" "https://percona-charts.storage.googleapis.com/pmm-server-${PMM_SERVER_VERSION}.tgz"
+	if [ ! -z "$OPENSHIFT" ]; then
+		platform=openshift
+		oc create sa pmm-server -n "${NAMESPACE}"
```

Comment:
Generally, with some very few exceptions, there is no need to use `${VARIABLE}` in Bash. Just `$VARIABLE` is enough, avoids clutter and makes the code easier to read.